src/components/Utils/Validate.js

```diff
@@ -4,8 +4,10 @@ const Required = (value) => {
   return 'Required!';
 };
 
-const RequiredInput = () => {
-  return false;
+const RequiredRefNumber = (value, initialValues) => {
```

Comment:
`initialValues` is a bad name, it doesn't mirror what's in there. Per [docs](https://redux-form.com/7.4.2/examples/fieldlevelvalidation/) it contains `allValues`